magma/conversions.py

```diff
@@ -200,30 +203,34 @@ def convertbits(value, n, totype, checkbit):
             Direction.InOut: InOut
         }[T.direction](Bit)
 
-    value = totype[len(Ts), T](ts)
+    value = totype[len(Ts), T](ts, name=name)
     if n is not None and len(value) < n:
+        # TODO: The extended value isn't named, but perhaps we'd like to move
```

Comment:
yeah I think we want to move to making the conversion operators not do extensions. imagine it will break a lot right now so probably will be a sizeable chagne (and affects outside people).